src/client/components/app-core/layout.css

```diff
@@ -6,3 +6,24 @@
 html {
   box-sizing: border-box;
 }
+
+ol {
```

Comment:
do we want to set `list-style: none` to an `<ol>`, since its purpose is to indicate an order?